examples/hooks/index.tsx

```diff
@@ -53,7 +54,12 @@ const Title = () => {
 
 const App = () => {
   return (
-    <Router basePath="/hooks" history={myHistory} routes={appRoutes}>
+    <Router
+      basePath="/hooks"
+      history={myHistory}
+      routes={appRoutes}
+      plugins={[createResourcesPlugin({})]}
```

Comment:
Is this a bad pattern? Would this create a new plugin instance on every re-render?

Reply:
good point

Reply:
Fixed